pkg/registry/admissionregistration/validatingadmissionpolicybinding/authz.go

```diff
@@ -85,26 +85,60 @@ func (v *validatingAdmissionPolicyBindingStrategy) authorize(ctx context.Context
 		}
 	}
 
+	var attrs authorizer.AttributesRecord
 	paramRef := binding.Spec.ParamRef
 
-	// require that the user can read (verb "get") the referred resource.
-	attrs := authorizer.AttributesRecord{
-		User:            user,
-		Verb:            "get",
-		ResourceRequest: true,
-		Name:            paramRef.Name,
-		Namespace:       paramRef.Namespace,
-		APIGroup:        apiGroup,
-		APIVersion:      apiVersion,
-		Resource:        resource,
+	//!NOTE: This applies a more restrictive requirement when using
+	// cluster-scoped params. Requires LIST of the resource paramKind.
+	// Can relax if we have access to discovery/restmapper to decide if
+	// paramKind is ClusterScoped so we can disambiguate the two cases.
+	switch {
+	case paramRef == nil:
+		// No authz check necessary
+	case len(paramRef.Name) > 0 && len(paramRef.Namespace) > 0:
```

Comment:
simplify this case to `len(paramRef.Name) > 0` ... if the name is specified, we're authorizing a `get`

Reply:
ah, did not realize you could supply empty namespace to mean wildcard auth. changed this case. This also makes the above note in comments irrelevant, so that is removed